src/ECU.cpp

```diff
@@ -132,28 +132,28 @@ void ECU::run() {
 }
 
 //ROUTES DATA (READS ID AND SENDS IT TO THE RIGHT FUNCTION)
-void ECU::route(SensorData* data) {
-    switch (data->getId()) {
-        case THROTTLE1_ID:
-            updateThrottle(data);
+void ECU::route() {
```

Comment:
switch() needs a default

Reply:
Added to be more clear. Although the default behavior for C++ is to skip the block if no case is matched which is what we want to happen.

Reply:
Oh I never new that - I've only seen examples that have default! Compiler will inline that, so doesn't hurt.